components/strava/sources/activity-created/activity-created.mjs

```diff
@@ -1,23 +1,30 @@
-const strava = require("../../strava.app.js");
+import strava from "../../strava.app.mjs";
 
-module.exports = {
+export default {
   key: "strava-activity-created",
-  name: "Activity Created",
-  description: "Emits an event when a new activity is created",
-  version: "0.0.1",
+  name: "New Activity Created Event",
+  description: "Emit new event when a new activity is created",
+  version: "0.0.2",
+  type: "source",
   props: {
     strava,
     stravaApphook: {
+      //label not supported for prop stravaApphook
+      //description not supported for prop stravaApphook
```

Comment:
I think you can remove these two comments from here. This same suggestion applies for the other source components

Reply:
done